cmd/controller-manager/main.go

```diff
@@ -187,3 +214,13 @@ func main() {
 
 	select {}
 }
+
+type workloadEventRelay struct {
+	statusController *status.Controller
+}
+
+func (wer *workloadEventRelay) HandleWorkloadObjectEvent(gvr schema.GroupVersionResource, oldObj, obj util.MRObject, eventType binding.WorkloadEventType, wasDeletedFinalStateUnknown bool) {
```

Comment:
Missing documentation on exported function.

Reply:
I updated the source around here to make it clear that this is implementing `binding.WorkloadEventHandler`.

I do not favor just copying comments from interface to implementation, that does not add information. I have added some comments on the interface and the implementation type, hopefully it is clearer now.